### PR TITLE
updates README with status fields

### PR DESCRIPTION
**Pull request by DhritiShikhar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -273,11 +273,13 @@ spec:
 
 #### ServiceBindingRequestStatus
 
-`ServiceBindingRequestStatus` depicts the status of the Service Binding operator. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#spec-and-status
+`ServiceBindingRequestStatus` depicts the status of the Service Binding operator. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```
Comment (ron1) [inline README.md]: Doesn't the ServiceBindingRequestStatus depict the status of the Service Binding Request and not the Service Binding operator? The operator can support multiple ServiceBindingRequests. It seems unusual that the status of each ServiceBindingRequest CR would somehow contribute to the overall status of the operator.

File: README.md [hunk 2]
```diff
@@ -273,11 +273,13 @@ spec:
...
 
-| Field | Description |
-|-------|-------------|
-| BindingStatus | The binding status of Service Binding Request |
-| Secret | The name of the intermediate secret |
+```
+status:
+  bindingStatus: success
+  secret: binding-request
```
Comment (ron1) [inline README.md]: In order to support multiple ServiceBindingRequests per namespace, is a unique name assigned to each binding-request intermediate secret?

File: README.md (hunk 1 above)
Comment (sbose78) [reply README.md]: Yes the `status` depicts the status of the specific request. A failure might indicate that the binding actors were missing, for example.

File: README.md (hunk 2 above)
Comment (sbose78) [reply README.md]: Yes, this is the name of the unique secret that gets created for the specific request. 

( need to check, there needs to be a fix to ensure a unique name is picked while creating the intermediate secret..)

**Comment by sbose78** [comment]
The CR could be in the readme itself. CC @pmacik .

File: README.md [hunk 3]
```diff
@@ -273,11 +273,13 @@ spec:
...
 
-| Field | Description |
-|-------|-------------|
-| BindingStatus | The binding status of Service Binding Request |
-| Secret | The name of the intermediate secret |
+```
+status:
+  bindingStatus: success
+  secret: binding-request 
+  applicationObjects: [binding-request]
```
Comment (sbose78) [inline README.md]: @DhritiShikhar doesn't look right to me :thinking:

**Comment by sbose78** [comment]
Dir structure has changed. This is now stale.

**Pull request closed without merging**